.github/workflows/ci.yml

```diff
@@ -28,20 +28,33 @@ jobs:
 
     - name: Test bin/run.sh and bin/runner
```

Comment:
```suggestion
    - name: Test bin/run.sh and bin/runner (Smoke test)
```

Reply:
Good idea. It's nice to distinguish this from our other tests, which are closer to unit tests. I've tried to clarify it a bit.